pkg/payment/request.go

```diff
@@ -247,6 +250,18 @@ type RulesRequest struct {
 	Discounts []DiscountRequest `json:"discounts,omitempty"`
 }
 
+// AuthenticationRequest represents payment authentication request within DataRequest.
+type AuthenticationRequest struct {
+	Type                 string `json:"type,omitempty"`
+	Cryptogram           string `json:"cryptogram,omitempty"`
+	ThreeDsServerTransId string `json:"three_ds_server_trans_id,omitempty"`
+	Eci                  string `json:"eci,omitempty"`
+	DsTransId            string `json:"ds_trans_id"`
+	AcsTransId           string `json:"acs_trans_id,omitempty"`
+	ThreeDsVersion       string `json:"three_ds_version,omitempty"`
```

Comment:
acho que 3DS, ECI e ACS são acrônimos
```suggestion
	ThreeDSServerTransID string `json:"three_ds_server_trans_id,omitempty"`
	ECI                  string `json:"eci,omitempty"`
	DSTransID            string `json:"ds_trans_id"`
	ACSTransID           string `json:"acs_trans_id,omitempty"`
	ThreeDSVersion       string `json:"three_ds_version,omitempty"`
```

Reply:
Corrigido!